### ISSUE TITLE
Potential way to show changelog

### ISSUE THREAD

**Issue by DericHuynh** [remyroy/CDDA-Game-Launcher#564]
**Problem:**
It should be clear that the problem with the new build system is that it doesn't seem to have an easy way of giving you the changelog of all commits between the current and previous version. It only shows you the commit that triggered the build for the current version.

**Useful Links**
Here is a list of the releases.
https://github.com/CleverRaven/Cataclysm-DDA/releases

Here is a list of the commits on master.
https://github.com/CleverRaven/Cataclysm-DDA/commits/master

**Solution:**
Goto https://github.com/CleverRaven/Cataclysm-DDA/releases and notice the Commit UID

![Example2](https://user-images.githubusercontent.com/54043695/119061485-b996db00-b991-11eb-8857-1058f303a622.png)

If you click on the commit UID link, you can get to the page of the specific commit that triggered the build. See this?

![ExampleOne](https://user-images.githubusercontent.com/54043695/119061312-4f7e3600-b991-11eb-9836-0fafca43712c.png)

That is a link to the previous commit this commit was built off of, and by creating a loop that continuously goes through each parent node until you reach the node of the previous build. This can be checked by comparing the commit UID of the node you're currently one and the commit UID of the previous release.

![Example2 - Copy](https://user-images.githubusercontent.com/54043695/119061702-3629b980-b992-11eb-9426-05690a512b52.png)

**Comment by remyroy** [comment]
Good idea. I don't see the launcher doing this for every user though. I would much rather see a centralized or hosted service doing this so the launcher could pull from there.

**Comment by ZhilkinSerg** [comment]
Changelog should be written in release notes. Possible approach is rather obvious - show list of commit descriptions between current and previous build. Somebody just needs to work on this.

**Comment by Cenness** [comment]
>That is a link to the parent node, the previous commit this commit was built off of, and by creating a loop that continuously goes through each parent node until you reach the node of the previous build, you can add the title of each commit to a list to build a changelog.

There is a `git log` command for that, no need to manually loop over. It even accepts tags:
```
# git log --oneline cdda-experimental-2021-05-22-0734..cdda-experimental-2021-05-23-0732
32f7fafc5f (tag: cdda-experimental-2021-05-23-0732) Merge pull request #48997 from BrettDong/osx-build
47b0658754 (tag: cdda-experimental-2021-05-23-0527) Merge pull request #48958 from eltank/no_leatherworking
1e7a8060a9 update unit test for crafting helmet_scavenger
dedfbe980c Fix compile error on macOS with l10n disabled
f2f07b93e6 Remove leatherworking proficiency from recipes that shouldn't need it
```
